Realm/ObjectServerTests/SwiftObjectServerTests.swift

```diff
@@ -2091,6 +2115,65 @@ class SwiftMongoClientTests: SwiftSyncTestCase {
     }
 }
 
+class AnyRealmValueSyncTests: SwiftSyncTestCase {
+    /// The purpose of this test is to confirm that when an Object is set on a mixed Column and an old
+    /// version of an app does not have that Realm Object / Schema we can still access that object via
+    /// `AnyRealmValue.dynamicSchema`.
+    func testMissingSchema() {
+        do {
+            let user = try logInUser(for: basicCredentials())
+
+            if isParent {
```

Comment:
Similar note as above where putting the child block first makes it a bit easier to follow.